tests/test_primary.py

```diff
@@ -251,7 +317,10 @@ def test_05_register_new_secondary(self):
 
   def test_10_register_ecu_manifest(self):
 
-    TestPrimary.instance.register_new_secondary('ecu11111')
+    # Throughout this function, I'll use a different nonces in each call to
+    # register_ecu_manifest, and check that the ones in calls expected to
+    # succeed have been noted and that the ones in calls expected to fail have
+    # not been noted.
 
     # TODO: Test providing bad data.
 
```

Comment:
I think the TODO on line 327 can be removed, given that the test conditions do indeed now use DER or JSON depending on `tuf.conf.METADATA_FORMAT`.

Reply:
Very true, thanks, will do. I think I'll also clarify 322.